spec/services/google_scraping_service_spec.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'vcr'
```

Comment:
Not sure why do we need to required vcr here, that was done in the rails_helper file as I remember

Reply:
Good catch. Fixed in 93579c70d309dc5d2a077f52b2b33bf088d14398